java/runtime/src/main/java/org/ray/runtime/runner/RunManager.java

```diff
@@ -182,7 +182,7 @@ public void startRayProcesses(boolean isHead) {
     }
   }
 
-  private void startRedisServer() {
+  private void startGcs() {
```

Comment:
Better to separate 2 methods:
1. startRedisServer
2. startGcsServer

Reply:
Gcs contain Redis and Gcs Server, so start them in one method is ok now.